openshift-knative-operator/cmd/operator/kodata/ingress/1.14/kourier/1-net-kourier.yaml

```diff
@@ -97,15 +97,6 @@ data:
                                       exact: GET
                               route:
                                 cluster: service_stats
-                            - match:
```

Comment:
@skonto what is the status of your kourier graceful shutdown testing?

Reply:
I need to do a PR against net-kourier https://github.com/openshift-knative/net-kourier/pull/130#issuecomment-2084532432 cherry-pick failed.